AppHubDeploy.js

```diff
@@ -24,6 +24,7 @@ program
   .option('-d, --build-description <description>', 'Description of the build. Wrap in quotes if more than one word.')
   .option('-o, --open-build-url',                  'Open AppHub Builds URL after a successful build and deploy.')
   .option('-n, --build-name <name>',               'Name of the build. Wrap in quotes if more than one word.')
+  .option('-p, --plist-file <plist>',              'Use a custom plist file for the `apphub` command.')
```

Comment:
Let's be as explicit as possible and use `<plist-file>`. 

Also, let's say `apphub build` in the description.